idea_plugin/google-java-format.iml

```diff
@@ -13,6 +13,6 @@
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
-    <orderEntry type="library" name="google-java-format-1.4-all-deps" level="project" />
+    <orderEntry type="library" name="google-java-format-1.5-SNAPSHOT-all-deps" level="project" />
```

Comment:
I admit it's not clear to me why google-java-format was "upgraded" to a SNAPSHOT version. Can you explain your reasoning to me here?